### PR TITLE
Add mobile number to registration form 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@justinvdm ready for review :)

File: tuneme/tests/test_views.py [hunk 1]
```diff
@@ -12,13 +12,14 @@
 from django.contrib.sites.models import Site
 
 
-class ViewsTestCase(TestCase, MoloTestCaseMixin):
+class ViewsTestCase(MoloTestCaseMixin, TestCase):
```
Comment (justinvdm) [inline tuneme/tests/test_views.py]: Why did we need to swap these around?

File: tuneme/templates/profiles/viewprofile.html [hunk 2]
```diff
@@ -17,6 +19,15 @@
         <h5>{{ alias }}</h5>
         <p>{% trans "Type new display name:" %}</p>
         {{ edit_profile_form.alias }}
+        {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
+          <h5>{% trans "Update your mobile number:" %}</h5>
+          <p>{% trans "(e.g. +260979000001)" %}</p>
+          <p>{% trans "Your mobile number is:" %}</p>
+          <h5>{{ mobile_number | default:''  }}</h5>
+  	      {{ edit_profile_form.mobile_number.errors }}
```
Comment (justinvdm) [inline tuneme/templates/profiles/viewprofile.html:27]: If this is the view page (not the edit page), why would there be errors here?

Comment (justinvdm) [inline tuneme/templates/profiles/viewprofile.html:27]: Ah, nvm, this page is for editing.

File: tuneme/static/css/styles.css [hunk 3]
```diff
@@ -209,6 +209,9 @@ button.smiley.happy:before{border: 0!important;}
 /* Footer */
 .footer {padding:15px; font-size:85%; clear:both; display:block;}
 
+.terms_and_conditions { text-align: left; }
```
Comment (justinvdm) [inline tuneme/static/css/styles.css]: Should we use dashes here to be consistent with the other class names (so `.terms-and-conditions`).

**Comment by justinvdm** [comment]
Two minor questions, looks good, :+1: once those are resolved.

**Comment by smarzban** [comment]
@justinvdm done, Thanks :)

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**